app/src/main/res/values/strings.xml

```diff
@@ -653,4 +654,11 @@ be temporary, and you can try again later.</li>
     This is displayed during the authentication flow as the app launches.-->
     <string name="biometric_auth_moved_too_quickly">Finger moved too fast. Try again.</string>
 
+    <string name="enable_search_suggestion_title">Would you like to turn on search suggestions?</string>
+    <string name="enable_search_suggestion_subtitle">To get search suggestions, %1$s needs to send what you type in the address bar to the search engine you\'ve chosen. %2$s.</string>
+    <string name="enable_search_suggestion_subtitle_learnmore">Learn more</string>
+    <string name="enable_search_suggestions_no">NO</string>
+    <string name="enable_search_suggestions_yes">YES</string>
```

Comment:
Please just use "normal case" in the Strings (Yes, No). Then on your View use `android:textAllCaps="true"`. For some views like `Button` Android's default theme may already do that.